src/ProgressOnderwijsUtils/Extensions/EnumerableOfStringExtension.cs

```diff
@@ -26,7 +25,7 @@ public static string JoinStrings([NotNull] this IEnumerable<string> strings)
         /// <param name="separator">separator string</param>
         /// <returns>a string</returns>
         [NotNull]
-        public static string JoinStrings([NotNull] this IEnumerable<string> strings, string separator)
+        public static string JoinStrings([NotNull] this IEnumerable<string?> strings, string separator)

```

Comment:
is dit nodig? kunnen we niet gewoon zeggen dat we dat niet eten?

Reply:
string.Join accepteert in elk geval wel null